roles/sap_hana_install/defaults/main.yml

```diff
@@ -11,6 +11,10 @@ sap_hana_install_software_directory: '/software/hana'
 # created again before the extraction starts.
 sap_hana_install_software_extract_directory: "{{ sap_hana_install_software_directory }}/extracted"
 
+# If there is a requirement to cleanup "sap_hana_install_software_extract_directory" after SAP HANA Installation, then 
```

Comment:
Please remove the trailing spaces here and in the other lines in this file (where you split the lines).

Reply:
Ranjith, to make it faster and easier install pre-commit on your system: 
https://sap-linuxlab.github.io/contribute/__install_pre_commit/